app/services/gpt_service.py

```
@@ -0,0 +1,32 @@
+    
+    try:
+        response = client.responses.create(
+            model="gpt-4o-mini",
+            input = messages,
+            temperature = 0.5, # 생성된 텍스트의 무작위성을 결정
+            max_output_tokens = 200
+        )
+        print(response)
+        
+    except OpenAIError as e:
+        raise RuntimeError(f"GPT API 호출 실패: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)